x/btccheckpoint/types/genesis.pb.go

```diff
@@ -218,33 +218,33 @@ func init() {
 }
 
 var fileDescriptor_9776220697c13f63 = []byte{
```

Comment:
this file descriptor change does it matter for struct parsing? like will v1 babylon structures be able to be parsed correctly with this change `Babylon v2 `

Reply:
It shouldn't matter, it's just reflecting the addition of `v2` in the paths.